Ravel-Creators/Assets/RavelCreator/Scripts/Ravel/Configuration/NetworkConfig.cs

```diff
@@ -74,7 +74,7 @@ private string GetBaseUrl(AppMode mode, bool debugValue = false)
                     case AppMode.Test:
```

Comment:
line 13; REGION is unused

Reply:
Same as above, this is used in the non editor ravel.